### DISCUSSION TITLE
关于提升隧道限速的途径？

### DISCUSSION THREAD

**Discussion by chasiopen** [natfrp/wiki#233] [想法] [answered]
本人是MC腐竹，对于多人模组服来说，36兆的网速好像有点不太够，有没有计划付费购买增加限速上限呢？

**Comment by fakeboboliu** [comment, marked as answer]
您好，

我们暂时没有处理大带宽客户的能力，请考虑租用或托管服务器。
如需足够的消费弹性，也可考虑使用云服务商，如 UCloud。

**Reply by Zha-Miku** [reply]
我加钱可以吗

**Reply by fakeboboliu** [reply]
> 我加钱可以吗

我们目前正在提供内测的带宽计费模式，这将允许您以一个合理的价格获得超过 36M 的限速。
您可以到 普通用户群/VIP群 中联系管理获取具体说明和开启方式。
